packages/server/api/src/app/ee/custom-domains/custom-domain.service.ts

```diff
@@ -1,23 +1,60 @@
 import { databaseConnection } from '../../database/database-connection'
 import { buildPaginator } from '../../helper/pagination/build-paginator'
 import { paginationHelper } from '../../helper/pagination/pagination-utils'
+import { getEdition } from '../../helper/secret-helper'
+import { cloudflareHostnameServices } from './cloudflare-api.service'
 import { CustomDomainEntity } from './custom-domain.entity'
 import {
     CustomDomain,
     CustomDomainStatus,
     ListCustomDomainsRequest,
 } from '@activepieces/ee-shared'
-import { apId, SeekPage } from '@activepieces/shared'
+import { logger } from '@activepieces/server-shared'
+import { ActivepiecesError, ApEdition, apId, ErrorCode, SeekPage } from '@activepieces/shared'
+
+export type SSLParams = {
```

Comment:
Same for this one, as it will be inline in the axios, and we need to remove it from all methods